ui/src/form.tsx

```diff
@@ -252,6 +266,8 @@ export const
       justify={m.justify || 'start'}
       align={m.align || 'center'}
       inset={m.inset}
+      // Checks for '1' or positive float value ending with 'px', 'rem' or 'vh'.
+      height={/^\d+(\.\d+)?(px|rem|vh)$|^1$/.test(m.height || '') ? m.height : undefined}
```

Comment:
This is unnecessary, the docs should be enough.
```suggestion
      height={m.height}
```